packages/framework/package.json

```diff
@@ -99,7 +99,9 @@
     "ts-node": "^10.9.2",
     "tsup": "^8.0.2",
     "typescript": "^5.3.3",
-    "vitest": "^1.2.1"
+    "vitest": "^1.2.1",
+    "zod": "^3.23.0",
+    "zod-to-json-schema": "^3.23.0"
```

Comment:
Each of the validation libraries we add will need an `x-to-json-schema` import to support the JSON Schema transformation to support the JSON-Schema transit layer for the Inputs-form rendering to work successfully.

This raises a question of who should bring this library - should it be included in the `@novu/framework` dependencies, or provided by the SDK consumer?

Reply:
I think it's a choice of a plugin, we might have the same problems with class validator, etc... So I think this can be done by the user?

Reply:
I have added a runtime `require('zod-to-json-schema')` that will throw if the required module is not present.